package MTV.Mithril;

message LexicalBlockList{
  repeated LexicalBlock lexical_block = 1;
}

message LexicalBlock{
  // These are the starting and ending addresses marking off the
  // block.
  required uint64 low_pc = 1;
  required uint64 high_pc = 2;
}
